babel.config.js.bak

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Mind reverting this?

Reply:
yeap sorry again, did it in a rush 2am, removed